src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -104,13 +108,23 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
   }
 }
 
-async function initializeConnection(options: Required<BlazorOptions>, circuitHandlers: CircuitHandler[], logger: ILogger): Promise<signalR.HubConnection> {
+async function getNewCircuitId(options: BlazorOptions): Promise<string> {
+  const response = await fetch(`${options.serviceUrl}/start`, {
+    method: 'POST',
+  });
+
+  const responseBody = await response.json() as { id: string };
+
+  return responseBody.id;
```

Comment:
In the TS-side code, could we rename all references to "circuit ID" to something like `circuitIdRequestToken`, or whatever its more accurate name now is? Otherwise it's hard to keep track of what aspect of the circuit ID we're sharing with the client.

Reply:
Fair enough